src/components/formInput.scss

```diff
@@ -1,42 +1,88 @@
-$labelhighlight: rgb(18, 18, 146);
-$errorHighlight: red;
-$textcolor: #333;
+$highlight: red;
 $borderColor: #ccc;
 $labelColor: #aaa;
-$input-spacing: 10px;
+$font:"Public Sans", sans-serif;
 
 .form-group {
   margin-top: 12px;
   display: inline-block;
   position: relative;
 }
-
-.input-field {
+@mixin inputField {
   margin: 10px;
-  width: 300px;
   height: 20px;
-  border-radius: 5px;
-  border: 3px solid $borderColor;
-  padding: 6px $input-spacing 6px $input-spacing;
+  border-radius: 8px;
+  border: 1px solid $borderColor;
+  padding: 16.5px 14px;
```

Comment:
There is not really a pixel with decimal value. Most browsers cant paint decimal pixels!